src/app.ts

```diff
@@ -89,6 +96,31 @@ function createMailto(url: string): void {
   )
 }
 
+function addCustomCss(windowElement: BrowserWindow): void {
+  const cssBaseFile = 'style.css'
```

Comment:
Regarding my previous comment about this, I wouldn't extract cssBaseFile since we are only using it in one place.

@timche Thoughts?

Reply:
Agree.